addons/test_website/controllers/main.py

```diff
@@ -119,3 +119,7 @@ def get_post_method(self, **kw):
     @http.route(['/get_post_nomultilang'], type='http', auth="public", methods=['GET', 'POST'], website=True, multilang=False, sitemap=False)
     def get_post_method_no_multilang(self, **kw):
         return request.make_response('get_post_nomultilang')
+
+    @http.route(['/test_route_1'], type='*', auth="public", methods=['GET', 'POST'], csrf=False)
+    def test_route_1(self, **kw):
```

Comment:
Maybe improve naming to `test_route_json_http` ?

And also add a small docstring what it does / returns.